packages/editor/src/lib/options.ts

```diff
@@ -111,4 +116,5 @@ export const defaultTldrawOptions = {
 	actionShortcutsLocation: 'swap',
 	createTextOnCanvasDoubleClick: true,
 	exportProvider: Fragment,
+	noteShapeResizeMode: 'none',
```

Comment:
wdyt about the default being `scale`? seems like it might be a nice thing to enable by default

Reply:
if not in the SDK, i think this would be interesting to turn on for dotcom

Reply:
We could, yeah. I'm not opposed to it, other than that it complicates the UI. I might try doing a single resize corner or something in a follower.